client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/protocol/Codec10.java

```diff
@@ -47,6 +47,12 @@ public void writeClientListenerParams(Transport transport, ClientListener client
       // No-op
    }
 
+   @Override
+   public void writeExpirationParams(Transport transport, long lifespan, long maxIdle, byte timeUnits) {
+      transport.writeVInt((int) lifespan);
```

Comment:
Maybe we should print a warning message when they use a value that is too large to be sent in the old protocols.